cumulus/ansible/tasks/volume.py

```diff
@@ -0,0 +1,137 @@
+from cumulus.celery import command
+from cumulus.ansible.tasks.inventory import simple_inventory
+
+from cumulus.ansible.tasks.utils import run_playbook
+from cumulus.ansible.tasks.utils import get_playbook_directory
+from cumulus.ansible.tasks.utils import get_callback_plugins_path
+from cumulus.ansible.tasks.utils import get_library_path
+
+import os
+from cumulus.ssh.tasks.key import _key_path
+
+
+@command.task
+def create_volume(profile, volume, secret_key, girder_callback_info):
+
+    playbook = os.path.join(get_playbook_directory(),
+                            'volumes', 'ec2', 'create.yml')
+
+    extra_vars = {
+        'girder_volume_id': volume['_id'],
+        'profile_id': profile['_id'],
+        'volume_name': volume['name'],
+        'volume_size': volume['size'],
+        'volume_zone': volume['zone'],
```

Comment:
Access keys provide access to all zones,  its true you can't attach a volume to a node in a different zone,  but if you wanted to create a volume first (before the cluster)  or wanted to add an existing volume to girder then you need to be able to specify a volume specific zone.